recipes/fastk/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "FASTK" %}
+{% set version = "1.0" %}
+{% set sha256 = "80d483baa74afe491ed5a9b7444c5600d66c0b35fef82d48e5129aa90fb99e17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abueg/FASTK/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true [osx]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  host:
+    - bzip2
+    - zlib
+    - libcurl==7.61.0
+    - make
+    - gxx_linux-64==5.4.0
+#    - {{ compiler('cxx') }}
+#    - {{ compiler('c') }}
+
+test:
+  files:
+    - test.fa
+  commands:
+    - FastK -k5 test.fa
+
+about:
+  license: Proprietary
```

Comment:
This looks like MIT

Reply:
yes looks like edited MIT now that I look at it, will edit, thank you!

Reply:
Hi. It feels like MIT but it's not like it. The text is quite different.I don't think it's right to say the licence _is_ MIT

Reply:
Very true. This was my fault and its a shame it is not MIT :(

We need to change that.